resdk/resolwe.py

```diff
@@ -293,16 +293,23 @@ def run(self, slug=None, input={}, descriptor=None,  # pylint: disable=redefined
         else:
             raise ValueError("Unexpected behaviour at get process with slug {}".format(slug))
 
-        # Upload files in basic:file fields
+        # Pre-process inputs
         try:
             for schema, fields in iterate_fields(input, process['input_schema']):
                 field_name = schema['name']
                 field_type = schema['type']
                 field_value = fields[field_name]
 
+                # Wrapp `list:` fields into list if they are not already
```

Comment:
I agree that exception messages in resdk are terrible and should be improved. Our server is quite advanced and usually reports the error in the response body. 

I thought about this briefly and this is how I would solve the problem of ugly exceptions. We should create our exception class (in, e.g., `exceptions.py`), pass it the slumber/requests exception, extract the information from request body, append it to the exception message and raise our exception. [Here](https://github.com/genialis/resolwe-bio-py/blob/9f1aec60e5853cbe0cc1b2b8cb05b7ddeeb6eb94/resdk/resources/base.py#L154) is one example of how I made the exception message more informative. We could abstract this with a custom exception class.

Now to the second point, should we wrap values in lists automatically or not? I would like to hear more opinions, @jkokosar?